addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComMessage.java

```diff
@@ -46,6 +47,15 @@
     byte[] decodeMessage() throws RFXComException;
 
     /**
+     * Procedure for converting RFXCOM value to openHAB command.
+     *
+     * @param valueSelector
+     *
+     * @return Openhab command.
+     */
+    Command convertToCommand(RFXComValueSelector valueSelector) throws RFXComException;
```

Comment:
I think you might need a [trigger channel](https://github.com/eclipse/smarthome/issues/1043#issue-133769124) for this case.

They are already [added ](https://github.com/eclipse/smarthome/pull/1936)to the ESH core, but I am not sure if they are 100 percent functional.